cmd/apichecker/apicheck.go

```diff
@@ -0,0 +1,1397 @@
+												return resp, err
+											}
+											result := r.MatchString(y.Val)
+											if result {
+												resp = append(resp, y.Val)
+											}
+										}
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+	return resp, nil
+}
+
+// htmlScrapeYobit gets the check string for Yobit Exchange
+func htmlScrapeYobit(htmlData *HTMLScrapingData) ([]string, error) {
+	var resp []string
+	temp, err := http.Get(htmlData.Path)
+	if err != nil {
+		return resp, err
+	}
+	tokenizer := html.NewTokenizer(temp.Body)
+	var case1, case2, case3 bool
+loop:
+	for {
+		next := tokenizer.Next()
+		switch next {
+		case html.ErrorToken:
+			break loop
+		case html.StartTagToken:
+			token := tokenizer.Token()
+			if token.Data == htmlData.TokenData {
+				for _, x := range token.Attr {
+					if x.Key == htmlData.Key {
+						switch x.Val {
+						case "n4":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								if string(tokenizer.Text()) == "v2" {
+									case1 = true
+								}
+							}
+						case "n5":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								tempStr := string(tokenizer.Text())
+								if tempStr == "v3" {
+									case2 = true
+									resp = append(resp, tempStr)
+								}
+							}
+						case "n6":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								tempStr := string(tokenizer.Text())
+								if tempStr != "v4" {
+									case3 = true
+								}
+								if case1 && case2 && case3 {
+									return resp, nil
+								}
+								resp = append(resp, tempStr)
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+	return resp, nil
+}
+
+// htmlScrapeLocalBitcoins gets the check string for Yobit Exchange
+func htmlScrapeLocalBitcoins(htmlData *HTMLScrapingData) ([]string, error) {
+	temp, err := http.Get(htmlData.Path)
+	if err != nil {
+		return nil, err
+	}
+	a, err := ioutil.ReadAll(temp.Body)
+	if err != nil {
+		return nil, err
+	}
+	r, err := regexp.Compile(htmlData.RegExp)
+	if err != nil {
+		return nil, err
+	}
+	str := r.FindString(string(a))
+	sha := crypto.GetSHA256([]byte(str))
+	var resp []string
+	resp = append(resp, crypto.HexEncodeToString(sha))
+	return resp, nil
+}
+
+// trelloCreateNewCard creates a new card on the list specified on trello
+func trelloCreateNewCard(fillData *CardFill) error {
+	params := url.Values{}
+	params.Set("idList", fillData.ListID)
+	if fillData.Name != "" {
+		params.Set("name", fillData.Name)
+	}
+	if fillData.Desc != "" {
+		params.Set("desc", fillData.Desc)
+	}
+	if fillData.Pos != "" {
+		params.Set("pos", fillData.Pos)
+	}
+	if fillData.Due != "" {
+		params.Set("due", fillData.Due)
+	}
+	if fillData.MembersID != "" {
+		params.Set("idMembers", fillData.MembersID)
+	}
+	if fillData.LabelsID != "" {
+		params.Set("idLabels", fillData.LabelsID)
+	}
+	err := sendAuthReq(http.MethodPost,
+		fmt.Sprintf(pathNewCard, params.Encode(), apiKey, apiToken),
+		nil)
```

Comment:
Should create a struct for this result, otherwise json.Unmarshal will always return an error on a nil interface